### DISCUSSION TITLE
Welcome contributors to Dev Library's discussions channel!

### DISCUSSION THREAD

**Discussion by techtrailhead** [google/devlibrary#217] [Announcements]
Discuss, ask, or share any story around your work on Dev Library.

**Comment by gabriel-TheCode** [comment]
Thank you @techtrailhead ! 😊

**Comment by bampakoa** [comment]
Many thanks to the Dev Library team for giving us the chance to showcase our work through this awesome website 👏👏

**Comment by bampakoa** [comment]
Hello all 👋 One feature that I would like to see is a map that shows all the authors per country (if this type of data is available) similar to the Google Developers Experts directory map. What do you think?

**Comment by techtrailhead** [comment]
That's a great idea, Aristeidis.
Let me talk to the team to get it chalked out.

Best,
Ankita

**Comment by rubenperegrina** [comment]
Hello everyone,

I'm not sure if this question should go here, sorry if it's not the right place.

I am publishing my first blog post on Google Developers Library on the 15th of November, I received a confirmation email, this email says that my blog post will be published in 7 working days.

Is there anything else I need to do? I can't see my post or my profile.
I just sent an email to library-google-dev@google.com to fix it, but they redirect me to the GitHub discussions.

Any help is appreciated.

**Comment by SsomsakTH** [comment]
> พูดคุย ถาม หรือแบ่งปันเรื่องราวเกี่ยวกับงานของคุณใน Dev Library
